components/mod-event/constants.ts

```diff
@@ -14,6 +14,9 @@ export const MOD_EVENTS = {
   EMAIL: 'tools.ozone.moderation.defs#modEventEmail',
   TAG: 'tools.ozone.moderation.defs#modEventTag',
   DIVERT: 'tools.ozone.moderation.defs#modEventDivert',
+  APPEAL: 'appeal',
```

Comment:
Just a vibe thing, but I expected to encounter more logic associated with this new constant.  So just double checking it's all handled 👍

Reply:
yep yep, just needed to rewire the $type re-assignment.